src/TicTacToe/Main.java

```diff
@@ -1,19 +1,46 @@
 package TicTacToe;
 
+
 public class Main {
 
     public static void main(String[] args) {
         Board board = new Board();
-        Player Jack = new Player("Jack",board,'X');
-        Player Bob = new Player("Bob",board,'O');
+        Player jack = new Player("Jack", board, 'X');
+        Player bob = new Player("Bob", board, 'O');
+        EndingConditions endingConditions = new EndingConditions(board);
+        Player actualPlayer = jack;
+        Input input = new Input(board);
+
+
+        while (true) {
+            int position = input.chosePosition();
+            while (input.isSignIn(position)) {
+                System.out.println("There is a sign in this field, chose free field");
+                position = input.chosePosition();
+            }
+
 
+            actualPlayer.move(position);
 
+            board.displayBoard();
+            if (endingConditions.isWin()) {
```

Comment:
You can move it to separate method checkEndingConditions.

Reply:
done